test/units/modules/network/f5/test_bigip_device_auth_radius.py

```diff
@@ -0,0 +1,142 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright: (c) 2018, F5 Networks Inc.
+# GNU General Public License v3.0 (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import os
+import json
+import pytest
+import sys
+
+if sys.version_info < (2, 7):
+    pytestmark = pytest.mark.skip("F5 Ansible modules require Python >= 2.7")
+
+from ansible.module_utils.basic import AnsibleModule
+
+try:
+    from library.modules.bigip_device_auth_radius import ApiParameters
+    from library.modules.bigip_device_auth_radius import ModuleManager
+    from library.modules.bigip_device_auth_radius import ArgumentSpec
+
+    # In Ansible 2.8, Ansible changed import paths.
+    from test.units.compat import unittest
+    from test.units.compat.mock import Mock
+
+    from test.units.modules.utils import set_module_args
+except ImportError:
+    from ansible_collections.f5networks.f5_modules.plugins.modules.bigip_device_auth_radius import ApiParameters
+    from ansible_collections.f5networks.f5_modules.plugins.modules.bigip_device_auth_radius import ModuleManager
+    from ansible_collections.f5networks.f5_modules.plugins.modules.bigip_device_auth_radius import ArgumentSpec
+
+    # Ansible 2.8 imports
+    from units.compat import unittest
+    from units.compat.mock import Mock
+
+    from units.modules.utils import set_module_args
```

Comment:
incorrect file was used, not templated, copy paste is unacceptable review the template and correct these